internal/awsservice/cloudwatchlogs.go

```diff
@@ -135,6 +135,22 @@ func IsLogGroupExists(logGroupName string) bool {
 	return len(describeLogGroupOutput.LogGroups) > 0
 }
 
+func GetLogStreams(logGroupName string) []types.LogStream {
+	describeLogStreamsOutput, err := CwlClient.DescribeLogStreams(ctx, &cloudwatchlogs.DescribeLogStreamsInput{
+		LogGroupName: aws.String(logGroupName),
+		OrderBy:      types.OrderByLastEventTime,
+		Descending:   aws.Bool(true),
+		Limit:        aws.Int32(10),
+	})
+
+	if err != nil {
+		log.Printf("failed to get log streams for log group: %s - err: %s", logGroupName, err)
```

Comment:
nit. `%s` is for strings but you're passing an error for the second arg. I always use `%v`, but am open to whatever.

Reply:
updated